index.bs

```diff
@@ -1444,6 +1446,34 @@ with parameter |value| is:
 
 </div>
 
+<pre class=simpledef>
+Capability: <dfn export>intercept file dialogs</dfn>
+Key: "<code>interceptFileDialogs</code>"
+Value type: boolean
+Description: Defines whether the file dialogs should be intercepted and not shown on screen.
```

Comment:
Worth mentioning the default is `true`?

Reply:
I think the default should be false.

Reply:
Then the 'matched capability serialization algorithm for the "interceptFileDialogs" capability' should return false by default.
```
1. If |value| is true, return [=success=] with data true.

1. Return [=success=] with data false.
```

Reply:
The actual default behavior is defined in the `set of sessions for which file dialog interception is enabled`:
```
1. If |session|'s [=intercept file dialogs=] capability is true, append |session| to |sessions|.
```
So I think it makes sense to mention the default behavior in the description.